operator/config/network-policies/deploy.yaml

```
@@ -0,0 +1,493 @@
+      protocol: TCP
+    - port: 8443
+      protocol: TCP
+    to: []
+  - ports:
+    - port: 2049
+      protocol: TCP
+    - port: 2049
+      protocol: UDP
+    to: []
+  ingress:
+  - from:
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)